docs/source/managed-federation/uplink.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Apollo Uplink
+sidebar_title: Uplink
+---
+
+When using [managed federation](./overview/), your federated gateway regularly polls an endpoint called **Apollo Uplink** for its latest supergraph schema and other configuration:
+
+```mermaid
+graph LR;
+  subgraph "Your infrastructure"
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  gateway([Gateway]);
+  end
+  subgraph "Apollo cloud"
+  registry{{Apollo Schema Registry}};
+  uplink{{Apollo Uplink}}
+  end
+  serviceA & serviceB -->|Pushes schema| registry;
+  registry -->|Updates config| uplink;
+  gateway -->|Polls for config changes| uplink;
+  class registry secondary;
+  class uplink secondary;
+```
+
+To maximize uptime, Uplink is hosted simultaneously at _two_ endpoints, one in GCP and one in AWS:
+
+* GCP: `https://uplink.api.apollographql.com/`
+* AWS: `https://aws.uplink.api.apollographql.com/`
+
+By default in versions of `@apollo/gateway` v0.45.0 and later, your gateway polls the GCP URL at a regular interval. Whenever a poll request fails, the gateway retries that request with the AWS URL. It continues swapping between the two URLs until the request succeeds, or until reaching the maximum number of retries.
+
+Even if a poll request fails all retries, the gateway continues polling at the next interval, and it continues using its most recently obtained configuration.
+
+> Verisons of `@apollo/gateway` prior to v0.45.0 don't support multiple Uplink endpoints and only use the GCP URL by default.
+
+## Configuring polling behavior
+
+You can configure the list of Uplink URLs your gateway uses, along with the number of retries it performs for a failed poll request.
```

Comment:
`experimental_pollInterval` can be set to >=10000 (10s or more). Do we want to talk about that here?

FWIW, I plan to kill this option soon in favor of `pollIntervalInMs` (doesn't exist yet), so might be worth waiting to document it.

Reply:
Great point, I'm going to make this a separate task and will leave out here for the sake of getting the PR out